Source/ZenLib/Ztring.cpp

```diff
@@ -1352,15 +1353,16 @@ Ztring& Ztring::Date_From_Seconds_1970 (const int64s Value)
 Ztring& Ztring::Date_From_Seconds_1970_Local (const int32u Value)
 {
     time_t Time=(time_t)Value;
-    #if _POSIX_C_SOURCE >= 1 || _XOPEN_SOURCE || _BSD_SOURCE || _SVID_SOURCE || _POSIX_SOURCE
+    #if defined(HAVE_GMTIME_R)
     struct tm Gmt_Temp;
-    struct tm *Gmt=localtime_r(&Time, &Gmt_Temp);
+    struct tm *Gmt=gmtime_r(&Time, &Gmt_Temp);
     #elif defined(_MSC_VER)
     struct tm Gmt_Temp;
-    errno_t localtime_s_Result=localtime_s(&Gmt_Temp , &Time);
-    struct tm* Gmt=localtime_s_Result?NULL:&Gmt_Temp;
+    errno_t gmtime_s_Result=gmtime_s(&Gmt_Temp , &Time);

```

Comment:
You replaced `localtime_r()` by `gmtime_s()` here, why?
Please use `#if defined(HAVE_LOCALTIME_R)` and `localtime_r()`.

Reply:
Thanks for catching, this was the result of some copy-pasting.